administration/src/bp-modules/components/CustomDatePicker.tsx

```diff
@@ -24,38 +30,42 @@ const CustomDatePicker = ({
   minDate,
   maxDate,
   disableFuture,
+  sideComponent,
 }: CustomDatePickerProps): ReactElement => {
   const { viewportSmall } = useWindowDimensions()
   const formStyle = viewportSmall ? { fontSize: 16, padding: '9px 10px' } : { fontSize: 14, padding: '6px 10px' }
   const textFieldBoxShadow =
     '0 0 0 0 rgba(205, 66, 70, 0), 0 0 0 0 rgba(205, 66, 70, 0), inset 0 0 0 1px #cd4246, inset 0 0 0 1px rgba(17, 20, 24, 0.2), inset 0 1px 1px rgba(17, 20, 24, 0.3)'
   return (
-    <DesktopDatePicker
-      views={['year', 'month', 'day']}
-      value={date}
-      format='dd.MM.yyyy'
-      slotProps={{
-        clearIcon: { fontSize: viewportSmall ? 'medium' : 'small' },
-        openPickerIcon: { fontSize: 'small' },
-        field: { clearable: true, onClear },
-        textField: {
-          placeholder: 'TT.MM.JJJJ',
-          error: !isValid,
-          spellCheck: false,
-          onBlur,
-          sx: {
-            width: '100%',
-            input: formStyle,
-            boxShadow: !isValid ? textFieldBoxShadow : undefined,
+    <StyledDiv>
+      <DesktopDatePicker
+        views={['year', 'month', 'day']}
+        value={date}
+        format='dd.MM.yyyy'
+        slotProps={{
+          clearIcon: { fontSize: viewportSmall ? 'medium' : 'small' },
+          openPickerIcon: { fontSize: 'small' },
+          field: { clearable: true, onClear },
+          textField: {
+            placeholder: 'TT.MM.JJJJ',
+            error: !isValid,
+            spellCheck: false,
+            onBlur,
+            sx: {
+              width: '100%',
+              input: formStyle,
+              boxShadow: !isValid ? textFieldBoxShadow : undefined,
+            },
           },
-        },
-      }}
-      localeText={deDE.components.MuiLocalizationProvider.defaultProps.localeText}
-      disableFuture={disableFuture}
-      minDate={minDate}
-      maxDate={maxDate}
-      onChange={onChange}
-    />
+        }}
+        localeText={deDE.components.MuiLocalizationProvider.defaultProps.localeText}
+        disableFuture={disableFuture}
+        minDate={minDate}
+        maxDate={maxDate}
+        onChange={onChange}
+      />
+      {sideComponent}
```

Comment:
The icon is sadly not correctly vertically centered/aligned:
![image](https://github.com/user-attachments/assets/b979298f-a383-4b7d-9277-c8cc95c4c183)

Reply:
looks like the parent is not flex

Reply:
but first check where we want to put it in the end (see other comment)